5-design-patterns/homework/src/interfaces.ts

```diff
@@ -0,0 +1,14 @@
+export interface ShipmentParams {
+    shipmentId: number;
+    weight: number;
+    fromAddress: string;
+    fromZipCode: string;
+    toAddress: string;
+    toZipCode: string;
+}
+
+export enum SHIPMENT_TYPE {
```

Comment:
I think you don't need it while you have separated class for each shipment type.